horace_core/sqw/PixelData/@PixelData/noisify.m

```diff
@@ -0,0 +1,58 @@
+function pix_out = noisify(obj, varargin)
+% This is a method of the PixelData class.
+% It is used to noisify this object's signal and variance arrays, paging as required.
```

Comment:
I think "as well as" rather than "rather". "noisify" specifies the random noise source. Also I have not yet learned to type the string without making it "nosify".

Reply:
If you want to say that it uses the noisify function fine (although that is then a comment restating the code). Most people won't understand what "noisify" means.

I want to know "what" this function does in a function comment, not how it does it: it adds random noise to my data.

Reply:
Noisify is not just add random noise; it also does things to the error bars. It is a defined package of items that modify the pixel data and use of its name sounds highly warranted.